nsqadmin/static/js/app_state.js

```diff
@@ -30,8 +31,15 @@ var AppState = Backbone.Model.extend({
         this.set('graph_interval', interval);
     },
 
-    url: function(url) {
-        return '/api' + url;
+    basePath: function(p) {
+        // if base path is / then don't prefix
+        var bp = this.get('BASE_PATH') == '/' ? '' : this.get('BASE_PATH');
+        // trim trailing /
+        return (bp + p).replace(/\/$/, '');
```

Comment:
Need to do a little testing but I _think_ there's a bug here when base path is `/` and this is passed a path `/`

Reply:
such as https://github.com/nsqio/nsq/pull/856/files#diff-6c66351008f5616a7bf2b9cc9859dbbfR27

Reply:
Actually, that link is to the Go template function, not the JS one, but I think my original point still stands.